tests/integration/local/start_lambda/test_start_lambda.py

```diff
@@ -184,13 +184,28 @@ def test_lambda_function_raised_error(self):
 
         print(response_data)
 
+        # determine the line number of the statement that will raise the error
+        statement_raising_exception = 'raise Exception("Lambda is raising an exception")'
+        statement_raising_exception_line_no = 0
+        with open("tests/integration/testdata/invoke/main.py", "r") as f:
+            for i, line in enumerate(f):
+                if line.strip() == statement_raising_exception:
+                    statement_raising_exception_line_no = i + 1
+        if not statement_raising_exception:
+            raise Exception(f'Cannot find statement "{statement_raising_exception}" in invoke/main.py')
```

Comment:
We should use `self.fail()` here instead of an exception.

Reply:
addressed